projects/plugins/jetpack/modules/masterbar/admin-color-schemes/colors/fresh/colors.scss

```diff
@@ -4,6 +4,13 @@
 @import "../_inline-help";
 @import "sidebar-notice";
 
+#wpadminbar .ab-icon,
```

Comment:
I don't think we need to update this... it's unused now 🙈

Reply:
I just want to keep it synced before we remove these unused files 🙂